Packs/rasterize/Integrations/rasterize/rasterize.yml

```diff
@@ -311,7 +311,7 @@ script:
     - contextPath: InfoFile.Type
       description: The type of the image/pdf file
       type: string
-  dockerimage: demisto/chromium:1.0.0.56296
+  dockerimage: demisto/chromium:115.0.5790.69036
```

Comment:
Is this right?

Reply:
I think there is a script that @JudahSchwartz wrote that changes the docker image name. please confirm

Reply:
confirmed. 115.0.5790 is the chrome version